cmd/tailscaled/tailscaled.go

```diff
@@ -163,6 +163,34 @@ func main() {
 	}
 }
 
+func trySynologyMigration(p string) error {
+	if distro.Get() != distro.Synology {
```

Comment:
if runtime.GOOS != "linux" || ....

Then this would func body will get compiled away on other platforms.

Reply:
oh! sweet i didn't know about this!